libs/core/schematics/migrations.json

```diff
@@ -0,0 +1,9 @@
+{
+    "schematics": {
+        "migration-0-37": {
+            "version": "0.37",
```

Comment:
Does not this mean that if user updates from anything below 0.37.0 then this schematic will be called, but we're currently on 0.36.3. Am I getting this right?

Reply:
This schematics update is a breaking change by itself, once we merge this pr `ng add` will start working differently and that's why I set version here to `0.37`, merge of this pr will promote version to `0.37`